_includes/selector.html

```diff
@@ -392,9 +392,9 @@
             packages: ["Standard", "Choose Specific Packages"],
             additional_pip_packages: ["cuDF", "dask-cuDF", "cuML", "cuGraph", "cuSpatial/cuProj", "cuxfilter", "cuCIM", "RAFT", "cuVS"],
             additional_rapids_packages: ["cuDF", "cuML", "cuGraph", "cuSpatial/cuProj", "cuxfilter", "cuCIM", "RAFT", "cuVS"],
-            additional_packages: ["JupyterLab", "Plotly Dash", "Graphistry", "TensorFlow", "Xarray-Spatial", "PyTorch"],
+            additional_packages: ["Graphistry", "JupyterLab", "NetworkX + nx-cugraph", "Plotly Dash", "PyTorch", "TensorFlow", "Xarray-Spatial"],
             note_prefix: "<i class='fas fa-info-circle text-blue'></i>",
-            rapids_meta_pkgs: ["cuDF", "cuML", "cuGraph", "cuSpatial", "cuProj", "cuxfilter", "cuCIM", "RAFT"],
+            rapids_meta_pkgs: ["cuDF", "cuML", "cuGraph", "cuSpatial", "cuProj", "cuxfilter", "cuCIM", "nx-cugraph", "RAFT"],
```

Comment:
We don't need both `NetworkX + nx-cugraph` and a separate button for `nx-cugraph`. Choose one or the other.

Reply:
Sorry, maybe I misunderstood this. I need to look at where this variable is used.

Reply:
I get it now. The `rapids_meta_pkgs` is a list of things in the `rapids` conda metapackage. Apologies, you did this right. I just didn't understand it at first.

Reply:
This doesn't add a separate button for `nx-cugraph`. This is for the disclaimer that tells users that the Standard RAPIDS metapackages comes with `nx-cugraph` 

![image](https://github.com/user-attachments/assets/264d70d5-c3e5-49c2-8b19-6fa4d857355e)